proto/birthday.proto

```diff
@@ -5,66 +5,72 @@ option go_package = "github.com/yarinBenisty/birthday-service/proto/birthday.pro
 package birthday;
 
 service BirthdayFunctions {
-    // CreateBirthday creates a new birthday and returns it.
-    rpc CreateBirthday(CreateBirthdayRequest) returns (BirthdayObject) {}
+  // CreateBirthday creates a new birthday and returns it.
+  rpc CreateBirthday(CreateBirthdayRequest) returns (BirthdayObject) {}
 
-    // GetBirthday returns the selected birthday.
-    rpc GetBirthday(GetBirthdayRequest) returns (BirthdayObject) {}
+  // GetBirthday returns the selected birthday.
```

Comment:
How does it find the birthday? (Using the personal number)
- If it is not part of the name (for examle, `GetBirthdayByPersonalNumber`), then you should document it.

Reply:
done